### PR TITLE
Apply feedback from Scrutinizer's informational recommendations

### PR DESCRIPTION
**Pull request by initvector**
Scrutinizer gives code feedback. This update applies most of the current "informational" suggestions.

### CODE REVIEW

File: applications/dashboard/modules/class.pagermodule.php [hunk 1]
```diff
@@ -64,14 +64,14 @@ class PagerModule extends Gdn_Module {
     public $Offset;
 
     /** @var int The last offset of the current page. (ie. Offset to LastOffset of TotalRecords). */
-    private $_LastOffset;
+    protected $_LastOffset;
```
Comment (DaazKu) [inline applications/dashboard/modules/class.pagermodule.php]: Why going from private to protected? We want attributes to be private by default.

File: plugins/Facebook/class.facebook.plugin.php [hunk 2]
```diff
@@ -13,7 +13,7 @@ class FacebookPlugin extends Gdn_Plugin {
     const API_VERSION = '2.7';
 
     /** Authentication table key. */
-    const ProviderKey = 'Facebook';
+    const PROVIDER_KEY = 'Facebook';
```
Comment (DaazKu) [inline plugins/Facebook/class.facebook.plugin.php]: I did not find any reference to ProviderKey so I'm inclined to accept that change but that's a bit risky.

File: applications/dashboard/modules/class.pagermodule.php (hunk 1 above)
Comment (initvector) [inline applications/dashboard/modules/class.pagermodule.php]: `MorePagerModule` extends `PagerModule`, but was including its own private `$_LastOffset` property. To ensure child classes can share the property, the access specifier was downgraded from private to protected.

File: plugins/Facebook/class.facebook.plugin.php (hunk 2 above)
Comment (initvector) [inline plugins/Facebook/class.facebook.plugin.php]: I've only found references to the `ProviderKey` class constants in older social plugins: Facebook, Google Plus and Twitter. In each case, the constant is only ever used internally within the class. That isn't to say that something, somewhere out there, isn't using these class constants for other purposes. However, it feels like a *relatively* safe change.

File: applications/dashboard/modules/class.pagermodule.php (hunk 1 above)
Comment (DaazKu) [inline applications/dashboard/modules/class.pagermodule.php]: I missed that haha. Scrutinizer is great!

**Pull request merged**